gpkit/solution_array.py

```diff
@@ -6,6 +6,53 @@
 from .small_scripts import unitstr, mag
 
 
+def senss_table(data, showvars=(), title="Sensitivities", minval=1e-2,
+                **kwargs):
+    "Returns sensitivity table lines"
+    if "constants" in data.get("sensitivities", {}):
+        data = data["sensitivities"]["constants"]
+    if showvars:
+        data = {k: data[k] for k in showvars if k in data}
+    return results_table(data, title, minval=minval, sortbyvals=True,
+                         valfmt="%+-.2g ", vecfmt="%+-8.2g",
+                         printunits=False, **kwargs)
+
+
+def topsenss_table(data, showvars, topN=5, **kwargs):
+    "Returns top sensitivity table lines"
+    data, filtered = topsenss_filter(data, showvars, topN)
+    title = "Most Sensitive" if not filtered else "Next Largest Sensitivities"
+    return senss_table(data, (), title, 0, **kwargs)
+
+
+def topsenss_filter(data, showvars, topN=5):
+    "Filters sensitivities down to top N"
+    if "constants" in data.get("sensitivities", {}):
+        data = data["sensitivities"]["constants"]
+    mean_abs_senss = {k: np.abs(s).mean() for k, s in data.items()
+                      if not np.isnan(s).any()}
+    topk = [k for k, _ in sorted(mean_abs_senss.items(), key=lambda l: l[1])]
+    filter_already_shown = showvars.intersection(topk)
+    for k in filter_already_shown:
+        topk.remove(k)
+        if topN > 3:  # always show at least 3
+            topN -= 1
+    return {k: data[k] for k in topk[-topN:]}, filter_already_shown
+
+
+def insenss_table(data, _, maxval=0.1, **kwargs):
+    "Returns insensitivity table lines"
+    if "constants" in data.get("sensitivities", {}):
+        data = data["sensitivities"]["constants"]
+    data = {k: s for k, s in data.items() if np.mean(np.abs(s)) < maxval}
+    return senss_table(data, (), "Insensitive Fixed Variables", 0, **kwargs)
+
+TABLEFNS = {"sensitivities": senss_table,
+            "topsensitivities": topsenss_table,
+            "insensitivities": insenss_table,
+           }
```

Comment:
oh this already exists -- should it have an `N` argument? That might be more intuitive to control than `maxval` in some cases.